yb-voyager/versions/yb-versions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": [
+    "2025.1.1.1-b1",
+    "2024.2.4.0-b89",
+    "2024.1.6.1-b2",
+    "2.20.10.0-b29",
+    "2.25.2.0-b359"
+  ],
+  "latest_stable": "2025.1.1.1-b1"
```

Comment:
I think version is `2025.1.1.0-b1`
And also you need to update the `ybversion/constants.go` for the Latest stable

Reply:
right, my bad